docs/en/03_apache-solr.md

```diff
@@ -0,0 +1,71 @@
+# Apache solr
+
+Apache Solr is a fulltext search engine, an aspect which is often used
+alongside this module. But more importantly for us, it has bindings to [Apache Tika](http://tika.apache.org/)
+through the [ExtractingRequestHandler](http://wiki.apache.org/solr/ExtractingRequestHandler) interface.
+This allows Solr to inspect the contents of various file formats, such as Office documents and PDF files.
+The textextraction module retrieves the output of this service, rather than altering the index.
+With the raw text output, you can decide to store it in a database column for fulltext search
+in your database driver, or even pass it back to Solr as part of a full index update.
+
+In order to use Solr, you need to configure a URL for it (in `mysite/_config/config.yml`):
+
+```yml
+SilverStripe\TextExtraction\Extractor\SolrCellTextExtractor:
+  base_url: 'http://localhost:8983/solr/update/extract'
+```
+
+> [!NOTE] In case you're using multiple cores, you'll need to add the core name to the URL
+> (e.g. `http://localhost:8983/solr/PageSolrIndex/update/extract`).
+> The ["fulltext" module](https://github.com/silverstripe-labs/silverstripe-fulltextsearch)
```

Comment:
```suggestion
> The ["fulltext" module](https://github.com/silverstripe/silverstripe-fulltextsearch)
```
though may be better to just remove the mention of this module as it's super outdated and no-longer supported.

Reply:
Removed

Reply:
Actually on second thought have left it in, minus the `-labs` account name

The fulltextsearch module does have a CMS 5 compatible module, though it's obviously on its last legs

That module started off life intending to be abstract, but basically just intended up just being "the solr4 module used on CWP"

I'll create a CMS 6 PR to remove this entire page after merge-up since solr is not at all supported at that point